rpc_jobs/standard_job_postmerge.yml

```diff
@@ -0,0 +1,132 @@
+- project:
+    name:       "gating-post-merge"
+    repo_url:   "https://github.com/rcbops/rpc-gating"
+    branch:     "master"
+    image:      "xenial"
+    scenario:   "functional"
+    action:     "test"
+    jira_project_key: "RE"
+    jobs:
+      - 'PM_{name}-{branch}-{image}-{scenario}-{action}'
+
+- job-template:
+    name: 'PM_{name}-{branch}-{image}-{scenario}-{action}'
+    project-type: pipeline
+    concurrent: true
+    FLAVOR: "performance1-1"
+    IMAGE: "Ubuntu 16.04 LTS (Xenial Xerus) (PVHVM)"
+    properties:
+      - build-discarder:
+          num-to-keep: "30"
+      - github:
+          url: "{repo_url}"
+    parameters:
+      - rpc_gating_params
+      - instance_params:
+          IMAGE: "{IMAGE}"
+          FLAVOR: "{FLAVOR}"
+          REGIONS: "{REGIONS}"
+          FALLBACK_REGIONS: "{FALLBACK_REGIONS}"
+      - string:
+          name: REPO_URL
+          default: "{repo_url}"
+          description: Url of the repo under test
+      - string:
+          name: BRANCH
+          default: "{branch}"
+          description: Branch of the repo under test
+    triggers:
+      - timed: "@daily"
+
+    dsl: |
+      library "rpc-gating@${{RPC_GATING_BRANCH}}"
+      properties([pipelineTriggers([githubPush()])])
+
+      env.STAGES="Allocate Resources, Connect Slave, Cleanup, Destroy Slave"
+
+      // Pass details about the job parameters through
+      // to the target environment so that scripts can
+      // use them to adapt behaviour.
+      env.RE_JOB_NAME = "{name}"
+      env.RE_JOB_IMAGE = "{image}"
+      env.RE_JOB_SCENARIO = "{scenario}"
+      env.RE_JOB_ACTION = "{action}"
+      env.RE_JOB_FLAVOR = "{FLAVOR}"
+      env.RE_JOB_TRIGGER = "PM"
+      env.RE_JOB_BRANCH = "{branch}"
+
+      // Not part of the published interface, used by this job later on
+      // to create failure tickets in the correct project.
+      JIRA_PROJECT_KEY = "{jira_project_key}"
+
+      // Apply a global three hour timeout
+      timeout(time: 6, unit: 'HOURS'){{
+        common.shared_slave() {{
+          pubcloud.runonpubcloud {{
```

Comment:
How is this going to work ... are we going to have different job-templates for different node types?  I thought the plan was to allow each project to dictate what node type they run on?

Reply:
@mattt416 All standard jobs, at this time, use a single-use slave. That slave could be any pubcloud instance with any flavor (ie onmetal is an option).

Reply:
Yep the RE-For-Projects page indicates the type of execution environment provided to each hook. If that environment is single use slave, then the flavour and image can be specified in the JJB project.

Reply:
OK ... I guess that's alright, since at present it's only the opsfabric jobs that don't need a single use slave.